app/Exceptions/Handler.php

```
@@ -62,4 +67,39 @@ protected function unauthenticated($request, AuthenticationException $exception)
 
         return redirect()->guest('login');
     }
+
+    /**
+     * Render an exception into an HTTP JSON response.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Exception  $exception
+     * @return \Illuminate\Http\Response
+     */
+    protected function json($request, Exception $exception) {
+        $code = 500;
+
+        if ($this->isHttpException($exception)) {
+            $code = $exception->getStatusCode();
+        }
+
+        if ($exception instanceof ValidationException) {
+            $code = 422;
+        }
+
+        $response = [
+            'error' => [
+                'code' => $code,
+                'message' => $exception->getMessage(),
```

We should probably [hide error details in this response](https://github.com/DoSomething/northstar/blob/d3f583dca778bce96cf83b87be136a29466b8983/app/Exceptions/Handler.php#L139-L146) on production, so we don't accidentally expose sensitive information in error strings. (Also thinking that a standard "handler" makes sense as a package, but that's a task for another time!)